taker/src/main.rs

```diff
@@ -118,7 +118,9 @@ impl Opts {
             None => match network {
                 Network::Mainnet { .. } => MAINNET_MAKER.to_string(),
                 Network::Testnet { .. } => TESTNET_MAKER.to_string(),
-                Network::Signet { .. } => bail!("No maker default URL configured for signet"),
+                Network::Signet { .. } | Network::Regtest { .. } => {
+                    bail!("No maker default URL configured for signet")
+                }
```

Comment:
🔧

```suggestion
                Network::Signet { .. } | Network::Regtest { .. } => {
                    bail!("No maker default URL configured for {network:?}")
                }
```

And for the other cases too.